pkgs/development/python-modules/edk2-pytool-library/default.nix

```
@@ -17,7 +17,7 @@
 
 buildPythonPackage rec {
   pname = "edk2-pytool-library";
-  version = "0.23.2";
+  version = "0.23.3";
   pyproject = true;
 
   disabled = pythonOlder "3.11";
```

```suggestion
  disabled = pythonOlder "3.10";
```
according to https://github.com/tianocore/edk2-pytool-library/blob/v0.23.3/pyproject.toml#L12